src/coreclr/jit/emitriscv64.cpp

```diff
@@ -977,6 +991,165 @@ void emitter::emitIns_R_R_R(
     appendToCurIG(id);
 }
 
+bool emitter::tryEmitCompressedIns_R_R_R(
+    instruction ins, emitAttr attr, regNumber rd, regNumber rs1, regNumber rs2, insOpts opt)
+{
+    instruction compressedIns = tryGetCompressedIns_R_R_R(ins, attr, rd, rs1, rs2, opt);
```

Comment:
```suggestion
    assert(!emitComp->compGeneratingProlog && !emitComp->compGeneratingEpilog);
    instruction compressedIns = tryGetCompressedIns_R_R_R(ins, attr, rd, rs1, rs2, opt);
```
nit, just to be on the safe side:
> Here are some things that must be addressed in a follow up PR:
> * [...]
> * Use compressed instructions inside prolog & epilog and update the prolog length computation from unwind codes (refer to ARM32 implementation).